employee/read_only.c

```diff
@@ -6,23 +6,28 @@
 
 #include "../include/common.h"
 #include "include/employee.h"
+#include <assert.h>
 #include <stdint.h>
 #include <stdio.h>
 #include <stdlib.h>
 
-int main()
+int main(int argc, char *argv[])
 {
-	struct employee employee = {0, "Good", "Employee", 55000};
+	struct employee employee = {1000, "Good", "Employee", 55000.0};
 	double salary = 0.0;
+	printf("--- EMPLOYEE ---\n");
 	// set the permissions to read-only
 	struct employee *ro_employee = set_read_only(&employee);
-	inspect_pointer(ro_employee);
+	assert((cheri_perms_get(ro_employee) & CHERI_PERM_STORE) == 0);
 	print_details(ro_employee);
-	printf("The struct is read-only so trying to change the salary will make the program crash...");
+	printf("\nThe struct is read-only so trying to change the salary will make the program "
+		   "crash...\n");
```

Comment:
The indentation looks odd (with mixed tabs and spaces).

Reply:
This is done by `clang-format` automatically...

Reply:
Odd! Ok, never mind then.